packages/features/credentials/repositories/CredentialRepository.ts

```diff
@@ -293,4 +293,33 @@ export class CredentialRepository {
       },
     });
   }
+
+  async findByTeamIdAndSlugs({ teamId, slugs }: { teamId: number; slugs: string[] }) {
```

Comment:
Method used to get credentials that have integration attribute syncing enabled. For now it'll just be Salesforce